presto-spark-classloader-interface/src/main/java/com/facebook/presto/spark/classloader_interface/IPrestoSparkTaskExecutor.java

```diff
@@ -17,7 +17,7 @@
 
 import java.util.Iterator;
 
-public interface IPrestoSparkTaskExecutor
-        extends Iterator<Tuple2<MutablePartitionId, PrestoSparkMutableRow>>
+public interface IPrestoSparkTaskExecutor<T>
```

Comment:
Add a comment that `T` should only be `PrestoSparkMutableRow` or `PrestoSparkSerializedPage` when in use.

Reply:
Actually let me better add a marker interface and let the compiler do the job